internal/output/output_result.go

```diff
@@ -491,9 +491,6 @@ func processVulnGroups(vulnPkg models.PackageVulns) (map[string]VulnResult, map[
 	hiddenVulnMap := make(map[string]VulnResult)
 
 	for _, group := range vulnPkg.Groups {
-		slices.SortFunc(group.IDs, identifiers.IDSortFunc)
-		slices.SortFunc(group.Aliases, identifiers.IDSortFunc)
```

Comment:
One thing we need to confirm with the HTML output is that we want to show the CVE ID as the first alias. I know a CVE isn't really an alias of a USN or DSA, but we still want to display its CVE ID because many scanners default to printing CVEs as their vulnerability ID. We want to avoid confusing people when comparing our results with others. We can rename "Aliases" to something more reasonable, but just ensure the HTML output displays the CVE as the first item under the current "Aliases" column. (this is also why we want to print out aliases there)
![image](https://github.com/user-attachments/assets/58d3d4f6-defd-40f2-9747-884ca41525d6)